src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -41,12 +41,7 @@ public Iterable findAll() {
      */
     @GetMapping("/{id}")
     public Book findOne(@PathVariable Long id) {
-        try {
-            return bookRepository.findById(id).orElseThrow(BookNotFoundException::new);
-        } catch (BookNotFoundException e) {
-            throw new ResponseStatusException(
-                    HttpStatus.NOT_FOUND, "Book not found", e);
-        }
+        return bookRepository.findById(id).orElseThrow(BookNotFoundException::new);
```

Comment:
User services instead repositories in controllers, the card not mention that but is a good practice